viz_scripts/plots.py

```diff
@@ -370,11 +343,23 @@ def CO2_impact(x,y,color,plot_title,file_name):
             va='center',
             ha=ha, fontsize=12, color='black', fontweight='bold')
 
-        # map names to colors
+    # map names to colors
     cmap = {True: 'green', False: 'red'}
-
     patches = [Patch(color=v, label=k) for k, v in cmap.items()]
-
     plt.legend(labels=objects, handles=patches, loc='upper right', borderaxespad=0, fontsize=15, frameon=True)
-
-    plt.savefig(SAVE_DIR+ file_name, bbox_inches='tight')
+    plt.savefig(SAVE_DIR+file_name+".png", bbox_inches='tight')
+
+def timeseries_plot(x,y,plot_title,ylab,file_name):
+    fig, ax = plt.subplots(figsize=(16,4))
+    sns.lineplot(ax=ax, x=x, y=y).set(title=plot_title, xlabel='Date', ylabel=ylab)
+    plt.xticks(rotation=45)
+    plt.subplots_adjust(bottom=0.25)
```

Comment:
I'm going to merge this for now, but note that the other plots actually do all the data mapping and color selection in `plot.py`. So to be consistent, all the code around 

```
    "data = expanded_ct[['user_id','start_local_dt_year','start_local_dt_month','start_local_dt_day','Mode_confirm','Mode_confirm_EI(kWH)','Mode_confirm_lb_CO2','distance_miles']].copy()\n",
    "data.rename(columns={'start_local_dt_year':'year','start_local_dt_month':'month','start_local_dt_day':'day'}, inplace=True)\n",
    "data['date_time'] = pd.to_datetime(data[['year','month','day']])\n",
    "data = data.drop(columns=['year','month','day'])\n",
    "\n",
    "# Categorical type will include all days/modes in groupby even if there is no data for a particular tabulation\n",
    "data.user_id = pd.Categorical(data.user_id)\n",
    "data.date_time = pd.Categorical(data.date_time)\n",
```

should ideally be in here.

Again, I am not going to enforce this right now, we can clean it up in the next round.

Reply:
Recorded in https://github.com/e-mission/e-mission-docs/issues/788

Reply:
Added to cleanup issue
https://github.com/e-mission/e-mission-docs/issues/788